packages/analytics/src/index.ts

```diff
@@ -0,0 +1,81 @@
+import mixpanel, { Mixpanel } from 'mixpanel';
+import { mixpanelMock } from './mock/mixpanel';
+
+function bufferToHex(value: Buffer) {
+  return value.toString('hex');
+}
+
+class CentralizedAnalytics {
+  protected project: string;
+  protected driver: Mixpanel;
+  constructor(params: { project: string }) {
+    this.project = params.project;
+    this.driver =
+      process.env.NODE_ENV === 'test'
+        ? mixpanelMock
+        : mixpanel.init(process.env.MIXPANEL_TOKEN || 'secret');
+  }
+
+  createAccount(params: {
+    accountId: Buffer;
```

Comment:
Use `ObjectID` instead of `Buffer`. Check the package `object-id`. Apply this to all `Buffer` type parameters.